### PR TITLE
Server console error on Organization catalog page

### LINKED ISSUE #2918: Server console error on Organization catalog page
### Steps
1. Take the last develop
2. Sign in as Admin user
3. Create an Organization, connect anyAPI
4. Sign out
5. Navigate to Organization page

### Expected
No error

### Found
Error on a server console
```
Exception from sub organizationApisCount id e8MX2mTW5GPAWjNnM MongoError: $and/$or/$nor must be a nonempty array
  at Object.Future.wait (/home/daria/.meteor/packages/meteor-tool/.1.5.2.10g4bls++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:449:15)
  at [object Object]._.extend.count (packages/mongo/mongo_driver.js:1091:51)
```

@Vellames I think it is related to issue when you made API count for Organization card

### Steps
1. Take the last develop
2. Sign in as Admin user
3. Create an Organization
4. Sign out
5. Navigate to Organization page

### PR DESCRIPTION
**Pull request by Vellames**
Closes #2918 

Server console error on Organization catalog page fixed

### CODE REVIEW

**Comment by Vellames** [comment]
@marla-singer ready for review :grimacing:

**Comment by Vellames** [comment]
@ marla-singer One note:
I used `OrganizationApis.find (null);` to return an empty cursor, I also thought to use "this.next ()" instead. What do you think it's better?

File: apinf_packages/organizations/collection/server/publications.js [hunk 1]
```diff
@@ -76,7 +76,11 @@ Meteor.publish('organizationApisCount', function (organizationId) {
     });
 
     // Uses the apisSelector to get available APIs
-    organizationApisCount = OrganizationApis.find({ $or: apisSelector });
+    if (apisSelector.length > 0) {
+      organizationApisCount = OrganizationApis.find({ $or: apisSelector });
+    } else {
+      organizationApisCount = OrganizationApis.find(null);
```
Comment (marla-singer) [inline apinf_packages/organizations/collection/server/publications.js]: I found another solution

```js
// Iterates over api organizations to get the publics apis
    organizationApisCount.forEach((organizationApi) => {
      // Get API information
      const api = Apis.findOne({ _id: organizationApi.apiId });

      // Checks if the user is manager of this API or API is public
      if (api.isPublic || (userId && api.currentUserCanManage(userId))) {
        // Then add IDs in list
        apisSelector.push(api._id);
      }
    });

    // Uses the apisSelector to get available APIs
    organizationApisCount = OrganizationApis.find({ apiId: { $in: apisSelector } });
```

**Comment by Vellames** [comment]
@marla-singer done

**Pull request merged**